docs/source/models/gbd2017/causes/causes.rst

```diff
@@ -108,13 +108,41 @@ The Appendix for the GBD 2017 Cause of Death manuscript is also available and Op
 Modeling non-fatal outcomes (Outline)
 ---------------------------
 
-- Figure of process
 .. image:: gbd_nonfatal_flowchart_sim_team.jpg
    :width: 600
 
-**Modeling approach** 
-
-- Modeling tools like DisMod, ST-GPR
+**DisMod** 
+
+Disease Model Meta Regression 2.1 (DisMod, sometimes DisMod MR) is a statistical modeling tool developed for the Global
+Burden of Disease study to estimate non-fatal disease burden. It is the most frequently used tool for non-fatal modeling
+in the GBD and models that are run outside of it are frequently called *custom models*. 
+
+DisMod has a user interface. The 
+`External Version
+<https://vizhub.healthdata.org/epi/>`_
+shows published and final models for GBD 2017.
+The 
+`Internal Version
+<https://internal.ihme.washington.edu/epi/>`_
+is where new and ongoing modeling occurs and is for GBD 2019 (but shows GBD 2017 best models). 
+
+There are three main components to DisMod. The first is that it
+is a meta-regression statistical model. This means that it uses point estimates with uncertainty around those estimates and covariates
```

Comment:
Regarding covariates: I'm not sure when we'll switch from GBD 2017 to 2019--because I'm visualizing covariates as a dismod sub-section for 2017, and a crosswalk sub-section for 2019---but it seems relevant to mention the heterogeneity of most epi data in these models, and the role of covariates (/types of covariates) in making that data usable. Though I could also see an argument for keeping this section simple.